packages/cmr-client/tests/test-getConcept.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const sinon = require('sinon');
+const test = require('ava');
+const got = require('got');
+
+const getConceptMetadata = require('../getConcept');
+
+let statusCode;
+const stubclient = {
+  getCmrData: () => ({
+    statusCode,
+    body: JSON.stringify({
+      feed: {
+        entry: [{
+          time_start: '2017-10-24T00:00:00.000Z',
+          updated: '2018-04-25T21:45:45.524Z',
+          dataset_id: 'MODIS/Terra Surface Reflectance Daily L2G Global 250m SIN Grid V006',
+          data_center: 'CUMULUS',
+          title: 'MOD09GQ.A2016358.h13v04.006.2016360104606'
+        }]
+      }
+    })
+  })
+};
+
+test.serial('get CMR metadata, success', async (t) => {
+  statusCode = 200;
+  const stub = sinon.stub(got, 'get').callsFake(stubclient.getCmrData);
```

Comment:
Why are we stubbing `got` instead of using `nock`? It seems like we're trying to simulate if request success or failure, so I think mocking the response instead of the request library might be better

Reply:
Ok I can change it. I copy/pasted from the other test since I just moved the function.

Reply:
I have updated the tests